File: test/stack/longest-absolute-file-path.test.ts
import { lengthLongestPath } from '../../src/stack/longest-absolute-file-path';

describe('longest absolute file path', () => {
  test('longest absolute file path - test case 1', async () => {
    expect(lengthLongestPath('dir\n\tsubdir1\n\tsubdir2\n\t\tfile.ext')).toBe(20);
  });
});

File: test/leetcode/stack/longest_absolute_file_path_test.py
from leetcode.stack.longest_absolute_file_path import Solution


soln = Solution()


def test_case_1():
    assert soln.lengthLongestPath("dir\n\tsubdir1\n\tsubdir2\n\t\tfile.ext") == 20


def test_case_2():
    assert soln.lengthLongestPath("dir\n    file.txt") == 12
